lib/modules/manager/mix/artifacts.ts

```diff
@@ -26,15 +28,35 @@ export async function updateArtifacts({
   config,
 }: UpdateArtifact): Promise<UpdateArtifactsResult[] | null> {
   logger.debug(`mix.getArtifacts(${packageFileName})`);
-  if (updatedDeps.length < 1) {
-    logger.debug('No updated mix deps - returning null');
+  const isLockFileMaintenance = config.updateType === 'lockFileMaintenance';
+
+  if (is.emptyArray(updatedDeps) && !isLockFileMaintenance) {
+    logger.debug('No updated mix deps');
+    return null;
+  }
+
+  let lockFileName = getSiblingFileName(packageFileName, 'mix.lock');
+  let isUmbrella = false;
+
+  let existingLockFileContent = await readLocalFile(lockFileName, 'utf8');
+  if (!existingLockFileContent) {
```

Comment:
test `if (await localPathExists(lockfileName))` then return error while reading lockfile.

Reply:
Can you be more specific here? I'm having trouble to understand.

Reply:
you currently simply fallback to another umbrella check. the read local file function eats any errors and returns null. so you need to check at least if the file exists.

if it exists and content is null, the error out instead of silently doing the umbrella fallback inside the condition